### PR TITLE
UPDATE_SITE_PLUGIN action renamed as CONFIGURE_SITE_PLUGIN

### PR DESCRIPTION
**Pull request by oguzkocer**
`UPDATE_SITE_PLUGIN` and `UPDATE_SITE_PLUGIN_VERSION` was a little too confusing. This PR changes the `UPDATE_SITE_PLUGIN` action's name to `CONFIGURE_SITE_PLUGIN` and another PR will be coming to change `UPDATE_SITE_PLUGIN_VERSION` to `UPDATE_SITE_PLUGIN` since we don't need that `VERSION` part anymore with this change.

No implementation change has been made, but feel free to run the tests in `ReleaseStack_PluginTestJetpack` and `PluginSqlUtilsTest` which I've already done but just in case :)

/cc @aforcier

### CODE REVIEW

**Comment by oguzkocer** [comment]
I've realized that I missed something in the test which I've fixed in 988b057a6ca972e10df3ec7a4c9cd6fb6a76c07b

File: fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/plugin/PluginRestClient.java [hunk 1]
```diff
@@ -88,17 +88,17 @@ public void updateSitePlugin(@NonNull final SiteModel site, @NonNull final Plugi
...
                     }
                 },
                 new BaseErrorListener() {
                     @Override
                     public void onErrorResponse(@NonNull BaseNetworkError networkError) {
-                        UpdateSitePluginError updatePluginError = new UpdateSitePluginError(((WPComGsonNetworkError)
-                                networkError).apiError, networkError.message);
-                        UpdatedSitePluginPayload payload = new UpdatedSitePluginPayload(site, updatePluginError);
-                        mDispatcher.dispatch(PluginActionBuilder.newUpdatedSitePluginAction(payload));
+                        ConfigureSitePluginError updatePluginError = new ConfigureSitePluginError(((
```
Comment (aforcier) [inline fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/plugin/PluginRestClient.java]: We should rename the `updatePluginError` variable here too.

Comment (oguzkocer) [reply fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/plugin/PluginRestClient.java]: Fixed in 967b38c94699453e36704cc128094ee62c8b019b

**Comment by aforcier** [comment]
:shipit:

**Pull request merged**